errors/stack_trace.go

```diff
@@ -71,6 +80,16 @@ func GetRuntimeContext() (fileName, funcName string, line int) {
 	return
 }
 
+type stack []uintptr
+
+func (s *stack) StackTrace() errors.StackTrace {
```

Comment:
The fact that the type itself is not exported doesn't change the fact that the method can be called on that type. Unexported type just means you can not use the name of that type, but if you have a variable of that type you still can doe whatever is exported on that type. Also `:=` operator works as well for unexported types (e.g. `x := ConstructorOfUnexportedType`) since it can bypass the need for actually using the name of the type (`var x unexportedType` would be illegal).